extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -567,7 +589,7 @@ async def get(self, object_id: str = None):
                           data:
                             type: array
                             items:
-                              type: object
+                               type: object
```

Comment:
This looks like an accidental space (there should be two spaces)

Reply:
Good call.